spec/dap/proto/ldap_proto_spec.rb

```diff
@@ -237,8 +237,8 @@
     end
 
     context 'testing empty ASN.1 Sequence' do
-      hex = ['308400000000']
-      data = OpenSSL::ASN1.decode(hex.pack('H*'))
+
+      data = OpenSSL::ASN1::Sequence.new([])
```

Comment:
I admit I didn't ask/figure out what this hex was originally.  So, what is the difference between the empty sequence before and now?  IOW, what was the value of `data` when it passed under 2.3.x and what is it now when it passes under (presumably) both?